patches/squirrel.mac/feat_add_new_squirrel_mac_bundle_installation_method_behind_flag.patch

```diff
@@ -0,0 +1,128 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <samuel.r.attard@gmail.com>
+Date: Mon, 28 Mar 2022 02:43:18 -0700
+Subject: feat: add new Squirrel.Mac bundle installation method behind flag
+
+The 'SquirrelMacEnableDirectContentsWrite' user default in your apps defaults suite
+can be used to control this new installation method.  It is designed to remove the
+requirement that the updating process have write access to its parent directory.
+
+With this feature enabled the updating process only needs write access to it's own
+.app bundle folder, not the owning /Applications folder.  This should allow more
+non-admin users to update applications when appropriately granted group based permissions.
+
+E.g. 775 and chown :staff
```

Comment:
upstream?

Reply:
Honestly, we are the upstream at this point, to avoid potential issues I'll properly upstream this when it's not behind the flag anymore

Reply:
Any reason not to just PR into Squirrel.Mac and bump our DEPS?

Reply:
Idk who uses Squirrel.Mac directly anymore and I don't want to be responsible for them 😓

Reply:
... move the Squirrel.Mac source into e/e then? 😅